docker/Dockerfile.test

```
@@ -0,0 +1,104 @@
+FROM php:7.0-apache
```

You can specify [php:7.0.20-apache](https://hub.docker.com/layers/php/library/php/7.0.20-apache/images/sha256-4c3fd256fb062bc663b37c38bd31c45ef0344588a33ed234d01fdb3822900ced?context=explore) version (like production env)

```suggestion
FROM php:7.0.20-apache
```